lib/repository.py

```diff
@@ -56,6 +56,19 @@ def get_git_repository(remote_repo_url, parent_dir_path):
                                         proxy=CONF.get('http_proxy'))
 
 
+def get_svn_repository(remote_repo_url, repo_path):
+    """
+    Get a local subversion repository located in a directory.
+    If it does not exist, check it out from the specified URL.
+    """
+    if os.path.exists(repo_path):
+        return SvnRepository(remote_repo_url, repo_path)
+    else:
+        # TODO: setup HTTP proxy by editing ~/.subversion/servers
```

Comment:
Thanks for the info, I'll add it then.

My point is still valid. If a team requires using a proxy, they have never built gcc or they have already setup a proxy in their subversion configuration, as the current command used to download gcc source code (svn export) would not work without it.